### PR TITLE
Leave Join Community Return Community information

### PR DESCRIPTION
**Pull request by sotails**

### CODE REVIEW

File: tests/test_community.py [hunk 1]
```diff
@@ -202,11 +202,11 @@ def test_community_join_post(client):
             follow_redirects=True,
         )
         assert (
-            b'{"access_token":"RefreshToken","message":"Community Joined","refresh_token":"AccessToken","status_code":200}\n'
+            b'{"access_token":"RefreshToken","data":{"Members":[],"name":"mock"},"message":"Community Joined","refresh_token":"AccessToken","status_code":200}\n'
```
Comment (stephend017) [inline tests/test_community.py:205]: is this test case correct? the community returns with no members after a `join` request?

File: tests/test_community.py [hunk 2]
```diff
@@ -234,9 +234,9 @@ def test_community_leave_post(client):
         )
 
         assert (
-            b'{"access_token":"RefreshToken","message":"Community Left","refresh_token":"AccessToken","status_code":200}\n'
+            b'{"access_token":"RefreshToken","data":{"Members":[{"email":"Testemail@gmail.com","name":"Jeffery","token":"Toke"}],"name":"mock"},"message":"Community Left","refresh_token":"AccessToken","status_code":200}\n'
```
Comment (stephend017) [inline tests/test_community.py:237]: the members contains the entire user object? not just the user email (id)?

**Review by stephend017 — CHANGES_REQUESTED** [review]
Can you please address my comments before merging. Thank you

File: tests/test_community.py (hunk 1 above)
Comment (sotails) [reply tests/test_community.py:205]: Get is already mocked to have an empty members list, so when get is called to return again it pulls the empty member list

File: tests/test_community.py (hunk 2 above)
Comment (sotails) [reply tests/test_community.py:237]: From what it seems it stores email name and token. It should just have the email so I'll update that

File: tests/test_community.py (hunk 1 above)
Comment (stephend017) [reply tests/test_community.py:205]: okay, that makes sense but it kinda makes this test semi redundant right?

File: tests/test_community.py (hunk 2 above)
Comment (stephend017) [reply tests/test_community.py:237]: awesome thanks

File: tests/test_community.py (hunk 1 above)
Comment (sotails) [reply tests/test_community.py:205]: A bit, this is just to mainly verify the route works. The DB tests verify the update functionality

Comment (fatcat2) [reply tests/test_community.py:205]: We can fix that, but I think for the time that works

**Review by fatcat2 — APPROVED** [review]
looks ok, waiting for stephen

**Review by stephend017 — APPROVED** [review]
Good to go captain

**Pull request merged**